src/lib/search/search.ts

```diff
@@ -63,7 +63,14 @@ export function search(query: string) {
 				(a?.block?.breadcrumbs.length || 0) - (b?.block?.breadcrumbs.length || 0) || a.rank - b.rank
 			);
 		})
-		.map(({ block }) => block) as (Block & Show)[];
+		.map(({ block }) => {
+			return {
+				...block,
+				content: block?.content
+					// strip markdown braces but preserve the link text as well as the link, delimited by a hyphen
+					.replace(/\[([^\]]+)\]\(([^)]+)\)/g, '$1 - $2')
```

Comment:
not gonna lie copilot cooked this regex so would be ideal to have someone double check it.

Reply:
prolly fine right